projects/plugins/jetpack/modules/masterbar.php

```diff
@@ -39,7 +39,4 @@
  */
 if ( apply_filters( 'jetpack_load_admin_menu_class', false ) ) {
 	require_once __DIR__ . '/masterbar/admin-menu/load.php';
-
-	// Ensures Calypsoify does not modify the navigation.
-	add_filter( 'jetpack_calypsoify_override_nav', '__return_false' );
```

Comment:
This is not needed anymore cause we now rely only on the `calypsoify` param to enable calypsoify ONLY for gutenberg right?

https://github.com/Automattic/jetpack/pull/17826/files#diff-90ba6bc9b60f26f472da395ac3879ee3a0b6be4abb0b00cfc814fca221cfc433R67

cc @mmtr

Reply:
I think this is not needed now because the `should_override_nav` function has been removed. See https://github.com/Automattic/jetpack/pull/17826/files#diff-90ba6bc9b60f26f472da395ac3879ee3a0b6be4abb0b00cfc814fca221cfc433L91

It seems we no longer override the navigation

Reply:
Yes, that's right. As I saw it, that was a mechanism to selectively disable Calypsoify if nav unification was active, but now we're disabling it all the time unless we're in the block editor.